assets/js/components/consent-mode/ConsentModeSetupCTAWidget.js

```diff
@@ -203,10 +203,10 @@ function ConsentModeSetupCTAWidget( { Widget, WidgetNull } ) {
 			'dismiss_notification'
 		);
 
+		showTooltip();
+
 		// For the first two dismissals, we show the notification again in two weeks.
 		if ( dismissCount < 2 ) {
-			showTooltip();
```

Comment:
LGTM, but should we also do the same for ABR where this also exists?
https://github.com/google/site-kit-wp/blob/e6b528018ab63cbb482f4a7a3aa904664f3d2a45/assets/js/modules/adsense/components/dashboard/AdBlockingRecoverySetupCTAWidget.js#L181-L199

Reply:
Good shout, @aaemnnosttv. I've pushed the same fix for this widget too.